internal/querycoordv2/task/scheduler.go

```diff
@@ -279,11 +293,46 @@ func (scheduler *taskScheduler) Add(task Task) error {
 	}
 
 	scheduler.updateTaskMetrics()
+	scheduler.updateTaskDelta(task)
+
 	log.Ctx(task.Context()).Info("task added", zap.String("task", task.String()))
 	task.RecordStartTs()
 	return nil
 }
 
+func (scheduler *taskScheduler) updateTaskDelta(task Task) {
+	var delta int
+	var deltaMap map[int64]map[int64]int
+	switch task := task.(type) {
+	case *SegmentTask:
+		segment := scheduler.targetMgr.GetSealedSegment(task.CollectionID(), task.SegmentID(), meta.NextTargetFirst)
+		delta = int(segment.GetNumOfRows())
+		deltaMap = scheduler.segmentExecutingTaskDelta
+
```

Comment:
Maybe the segment task with `scope=streaming` since the row count may cause delegator score inaccurate